EIPS/eip-xx.md

```diff
@@ -0,0 +1,117 @@
+---
+eip: xx *number to be determined by editor*
+title: smart contracts in java
+author: jeremy akatsa (@jeyakatsa)
+description: java smart contract creation
+discussions-to: https://ethresear.ch/t/a-new-erc-token-proposal/11540
+status: draft
+type: standards track
+category: erc
+created: 2022-02-04
+---
+
+## Abstract
+Converting Solidity Keywords into Java Dependencies in order for a library of Smart-Contract implementations to be built in the Java programming language.
+
+## Motivation
+Currently, there are ***200,000 Solidity/Ethereum Developers (Worldwide*** *(source: [TrustNodes](https://www.trustnodes.com/2018/07/22/ethereums-ecosystem-estimated-200000-developers-truffle-seeing-80000-downloads-month))* ***)*** and ***7.1million Java Developers (Worldwide*** *(source: [Daxx](https://www.daxx.com/blog/development-trends/number-software-developers-world#:~:text=According%20to%20SlashData%2C%20the%20number,%2C6%20million%20(source)))* ***)*** respectfully.
+
+What if all those Java developers (currently in the millions), could be onboarded into the Ethereum Ecosystem?
+
+![](https://github.com/jeyakatsa/EIPs/blob/master/assets/eip-xx/Java-Abstraction-Visualization-display.jpg)
+
+## Specification
```

Comment:
An EIP should describe how the proposal should work. I don't view this specification as sufficient to converting a solidity program into java code or vice versa.

Reply:
Understood.